pkg/server/io/helpers_windows.go

```diff
@@ -0,0 +1,31 @@
+// +build windows
+
+/*
+Copyright The containerd Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io
+
+import (
+	"io"
+	"os"
+
+	"golang.org/x/net/context"
+)
+
+// TODO(windows): Add windows FIFO support.
```

Comment:
There is no such support. We have to do this with `npipe` unfortunately. Lets talk about this one before we pull the other code across.

Reply:
I have an implementation locally. I just mean we need to have the same functionality implemented in windows. It is not a FIFO of course. :)

Reply:
Ok cool!